android/src/main/kotlin/com/hiennv/flutter_callkit_incoming/CallkitNotificationManager.kt

```diff
@@ -186,7 +186,11 @@ class CallkitNotificationManager(private val context: Context) {
                     .into(targetLoadAvatarDefault)
             }
             val caller = data.getString(CallkitConstants.EXTRA_CALLKIT_NAME_CALLER, "")
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+            val notificationManager: NotificationManager =
+                context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+            val canUseFullScreenIntent =
+                Build.VERSION.SDK_INT < Build.VERSION_CODES.UPSIDE_DOWN_CAKE || notificationManager.canUseFullScreenIntent()
```

Comment:
This is incorrect. 

`Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE && notificationManager.canUseFullScreenIntent()`

Reply:
According to the documentation, apps always have the USE_FULL_SCREEN_INTENT permission on Android 14 or earlier. So canUseFullScreenIntent should be true on Android 14 or earlier.

`Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE && notificationManager.canUseFullScreenIntent()`
This code returns false on Android 14 or earlier.

https://developer.android.com/reference/android/app/NotificationManager#canUseFullScreenIntent()
```
From Android Build.VERSION_CODES.UPSIDE_DOWN_CAKE, apps may not have permission to use Manifest.permission.USE_FULL_SCREEN_INTENT
```

Reply:
I'm sorry.
I took this to mean whether this full-screen feature is enabled or not.

It's ok.